tests/subsys/edac/ibecc/src/ibecc.c

```diff
@@ -50,7 +50,7 @@ static void callback(const struct device *d, void *data)
 	error_syndrome = error_data->syndrome;
 }
 
-static void test_ibecc_api(void)
+static void test_ibecc_api_before_inject_test(void)
```

Comment:
why do you rename this function? This is API test. No need to add extra words here

Reply:
I rename it, because, these testing is valid only before injecting testing. After injecting tests, these testing API's returned values are not expected.